src/Core/Utility.re

```diff
@@ -894,3 +894,15 @@ module Json = {
     loop(json);
   };
 };
+
+module ColorEx = {
+  open Revery;
+
+  let mix = (~start, ~stop, ~amount) =>
+    Color.{
+      r: (stop.r -. start.r) *. amount +. start.r,
+      g: (stop.g -. start.g) *. amount +. start.g,
+      b: (stop.b -. start.b) *. amount +. start.b,
+      a: (stop.a -. start.a) *. amount +. start.a,
+    };
```

Comment:
Should probably be upstreamed to Revery

Reply:
Makes sense, this would be helpful in the `Color` module